job-dsls/src/main/resources/job-scripts/prod_create_release_trackers.jenkinsfile

```diff
@@ -0,0 +1,42 @@
+pipeline {
+    agent {
+        label 'kie-rhel7 && !master'
+    }
+
+    stages {
+        stage('Clean workspace') {
+            steps {
+                cleanWs()
+            }
+        }
+
+        stage('Print variables') {
+            steps {
+                println "[INFO] VERSION: ${VERSION}"
+                println "[INFO] TEMPLATE: ${TEMPLATE}"
+            }
+        }
+
+        stage('Clone the repository') {
+            steps {
+                sh "git clone ${INTEGRATION_PLATFORM_TOOLING_REPOSITORY}"
+            }
+        }
+
+        stage('Create the release tracker tickets') {
+            steps {
+                withCredentials([string(credentialsId: 'jira-token', variable: 'JIRA_TOKEN')]) {
+                    dir('integration-platform-tooling') {
+                        sh "python3 release-generator.py --headless --authtoken ${JIRA_TOKEN} ${VERSION} ${TEMPLATE}.yaml"
```

Comment:
is not the `python3` tool required?

Reply:
As python3 is installed via rpm and not downloaded via wget, tools section is not required.